content/guides/models/track/project-page.md

```diff
@@ -183,6 +180,10 @@ By default, this turns other numeric columns into histograms showing the distrib
 
 <!-- ## Automations tab -->
 
+## Automations tab
+Automate downstream actions for versioning artifacts. To create an automation, define trigger events and resulting actions. Actions include executing a webhook or launching a W&B job.
+
+{{< img src="/images/app_ui/automations_tab.png" alt="Automation tab that depicts two automations: a model CI/CD webhook and a hello-world launch job" >}}
```

Comment:
I wonder if it's best to omit this image, which features Launch even though Launch docs are not in an ideal state, or to instead show a Slack automation and a webhook automation. It's pretty simple to create a Slack automation using our instance.